tests/unit/net/fetch.rs

```diff
@@ -213,15 +213,20 @@ fn test_cors_preflight_fetch() {
     };
     let (mut server, url) = make_server(handler);
 
+    let target_url = url.clone().join("a.html").unwrap();
+    let desired_response_url = url.clone();
```

Comment:
The `url` will be moved to `request`, but I think it will be better to clone when building `request`.